src/components/reusable-ui/Card.jsx

```diff
@@ -0,0 +1,77 @@
+import styled from "styled-components"
+import { theme } from "../../theme/index.js"
+import PrimaryButton from "./PrimaryButton"
+
+export default function CardItem({ imageSource, title, price }) {
+  return (
+    <CarteItemStyled>
+      <div className="carte-img">
+        <img src={imageSource} alt="{title}" />
+      </div>
+      <h3>{title}</h3>
+      <div className="bottom-carte">
```

Comment:
```suggestion
      <div className="bottom-card">
```

Essayer d'éviter le mélange français anglais au sein du même nom de class. À rectifier pour le reste des noms de class.